packages/ses/NEWS.md

```diff
@@ -13,6 +13,11 @@ User-visible changes in SES:
   each evaluation will need a fresh `Compartment`.
 * BREAKING CHANGE: The `lockdown` function's deprecated `noTame*` options have
   been removed in favor of the `*Taming` options introduced in version 0.8.0.
+* BREAKING CHANGE: The `.toLocale*` methods of
```

Comment:
Should we also note that "we're about to restore them, so if you wait for the next release you don't need to worry about it"?